packages/api/endpoints/collections.js

```diff
@@ -113,50 +111,44 @@ async function get(req, res) {
  */
 async function post(req, res) {
   const {
-    collectionsModel = new models.Collection(),
     collectionPgModel = new CollectionPgModel(),
     knex = await getKnexClient(),
     esClient = await Search.es(),
   } = req.testContext || {};
 
   const collection = req.body || {};
-  const { name, version } = collection;
 
+  const { name, version } = collection;
   if (!name || !version) {
-    return res.boom.badRequest('Field name and/or version is missing');
+    return res.boom.badRequest(`Field name and/or version is missing in Collection payload ${JSON.stringify(collection)}`);
   }
+  validateCollection(collection);
```

Comment:
Should we move this after line 128? `createdAt` and `updatedAt` are required for collections and the body of the POST request might not include them, but it wouldn't matter because we're setting them manually

Reply:
Moved validateCollection() to after createdAt and updatedAt.